wayland-protocols-misc/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 0.2.0 -- 04/01/2023
```

Comment:
No, sorry if that was unclear, don't add a new header, I do that when doing the release. :)